src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx

```diff
@@ -31,57 +32,61 @@ const AccommodationsForm = (props: any): JSX.Element => {
         </SelectBoxWrapper>
       </div> */}
 
-      <div className="form-item row">
-        <div className="input-container">
-          <InputLabel htmlFor="numberOfBedrooms">Bedrooms</InputLabel>
-          <InputWrapper>
-            <Field
-              name="numberOfBedrooms"
-              placeholder="# of Bedrooms"
-              type="number" />
-          </InputWrapper>
-          <ErrorMessageWrapper>
-            <ErrorMessage name="numberOfBedrooms" />
-          </ErrorMessageWrapper>
-        </div>
-
+      <div className="form-item">
         <div className="input-container">
-          <InputLabel htmlFor="sleepingArrangement">Beds</InputLabel>
+          <InputLabel htmlFor="sleepingArrangement">Sleeping Arrangement</InputLabel>
           <InputWrapper>
             <Field
               name="sleepingArrangement"
-              placeholder="# of Beds"
+              placeholder="1 King, 2 Queens"
               type="text" />
           </InputWrapper>
         </div>
       </div>
 
-      <div className="form-item row">
-        <div className="input-container">
-          <InputLabel htmlFor="numberOfBathrooms">Bathrooms</InputLabel>
-          <InputWrapper>
-            <Field
-              name="numberOfBathrooms"
-              placeholder="# of Bathrooms"
-              step={0.5}
-              type="number" />
-          </InputWrapper>
-          <ErrorMessageWrapper>
-            <ErrorMessage name="numberOfBathrooms" />
-          </ErrorMessageWrapper>
+      <div className="form-item">
+        <div className="input-number-container">
+          <InputLabel>Number of Bedrooms</InputLabel>
+          <NumberInput
+            value={values.numberOfBedrooms}
+            max={50}
+            min={1}
+            onChange={(value: number) => {
+              setFieldValue('numberOfBedrooms', value);
+              setFieldTouched('numberOfBedrooms');
+            }}
+          />
```

Comment:
does this no longer need an error message because the user can't select a bad input?

Reply:
ya 👍